src/gui/properties/peerlistwidget.cpp

```diff
@@ -159,7 +159,10 @@ void PeerListWidget::showPeerListMenu(const QPoint&)
     QString myip = m_listModel->data(m_listModel->index(row, PeerListDelegate::IP_HIDDEN)).toString();
     QString myport = m_listModel->data(m_listModel->index(row, PeerListDelegate::PORT)).toString();
     selectedPeerIPs << myip;
-    selectedPeerIPPort << myip + ":" + myport;
+    if (myip.length() > 15) // IPv6
```

Comment:
Where does `15` come from? According to [this](https://en.wikipedia.org/wiki/IPv6#Address_representation) IPv6 addresses can be shortened so eg `::1` is a valid IPv6 address.
I propose searching for the presence of dots instead, which will denote IPv4.